src/class_singleCleaner.py

```diff
@@ -36,11 +36,12 @@
 
 
 class singleCleaner(StoppableThread):
+    """Module run singleCleaner thread"""
```

Comment:
Maybe "The singleCleaner thread class"? What does the module have to do with it?

Reply:
Nothing to do with the module. I write the wrong doc for this class. I will change the docstring and thank you for correcting me.